framework/components/ecl-context-navs/ecl-context-navs.twig

```diff
@@ -1,27 +1,53 @@
-<div class="context-nav">
-  <span class="context-nav__label">{{ label }}</span>
+{#
+  - "label" (string): label displayed before the list (default: '')
+  - "items" (array): [{
+    "href" (string): target of the link (default: '#')
+    "label" (string): label of the link (default: '')
+  }]
+  - "limit" (integer): max number of items to display (0 = display all) (default: 5)
+  - "more_label": label of the "see more items" button (default: "More ›")
+  - "extra_classes" (string): extra CSS classes to be added
+  - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+#}
+
+{% set label = label|default('') %}
+{% set limit = limit|default(4) %}
+{% set more_label = more_label|default('More ›') %}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-context-nav' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_class is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_Class %}
+{% endif %}
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~'"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result  #}
+
+<div class="{{ _css_class }}" {{ _extra_attributes }}>
+  <span class="ecl-context-nav__label">{{ label }}</span>
   {% if items is defined and items is iterable %}
-  <div class="context-nav__items">
+  <ul class="ecl-context-nav__list">
     {% for item in items %}
-    <span class="context-nav__item">
-      <a href="{{ item.target }}">{{ item.title }}</a>
-    </span>
+    <li class="ecl-context-nav__item{{ limit > 0 and loop.index > limit ? ' ecl-context-nav__item--over-limit' : '' }}">
+      <a class="ecl-link ecl-link--standalone" href="{{ item.href }}">{{ item.label }}</a>
+    </li>
     {% endfor %}
+
+    {% if items|length > limit %}
+    <li class="ecl-context-nav__item ecl-context-nav__more">
+      <span role="button" class="ecl-link ecl-link--standalone">{{ more_label }}</span>
```

Comment:
You can't reach the "more" link with the keyboard. Maybe it would make sense to use a button here?